source/crud/tests/unified/client-bulkWrite-qe.yml

```diff
@@ -0,0 +1,113 @@
+description: client bulkWrite with queryable encryption
```

Comment:
Update the bulk write spec to remove the [requirement to error if using Auto-Encryption](https://github.com/mongodb/specifications/blob/43d2c7bacd62249de8d2173bf8ee39e6fd7a686e/source/crud/bulk-write.md#auto-encryption).